### DISCUSSION TITLE
Do you use the Mill REPL? (`mill --repl`)

### DISCUSSION THREAD

**Discussion by lefou** [com-lihaoyi/mill#2034] [Polls]
Hi, I rarely use the Mill repl and from discussions and chats, I got the impression that most users and all Mill devs aren't using the repl at all. The Mill repl is provided by Ammonite, but for the future, we can't tell whether we will keep Ammonite as our foundation or not. 

So, here is my poll to find out, how many of you use the repl.

```bash
$ mill --repl
Loading...
Mill Build Tool, version 0.10.7
@ version()()
[1/1] version 
0.10.7
res2: String = "0.10.7"
@ ^D
Bye
$
```

**Comment by lihaoyi** [comment]
I never use the Mill REPL. I basically always use Mill from Bash, and find the inspection and debugging facilities provided by Mill's CLI to be enough for basically everything I need to do

**Comment by ckipp01** [comment]
I've actually never used `mill --repl` before. I answered `No, I use the shell`, but honestly I rarely use the REPL mainly because in my workflow I gravitate towards using worksheets more.

**Comment by lefou** [comment]
Thanks for your answers so far!

Looks like loosing the Mill REPL (not the `ScalaModule.repl` target) eventually is no big deal.

**Comment by hmf** [comment]
I don't use the REPL. Nearly always use `-i`
HTHs

**Comment by lefou** [comment]
Thanks for all your answers.

**Comment by lefou** [comment]
Mill 0.11.0 now no longer has a REPL.
